app/layout.tsx

```diff
@@ -6,7 +6,14 @@ import Footer from "./components/Footer"
 
 export const metadata: Metadata = {
 	title: "San Diego DEVx",
-	description: "All things Software Development"
+	description: "Fostering developer community through events and open-source projects.",
+	openGraph: {
+		type: "website",
+		url: "https://devx.network",
+		title: "San Diego DEVx",
+		description: "Fostering developer community through events and open-source projects.",
+		images: "/images/DEVxCover.png"
```

Comment:
I think this image url be absolute, or we could define a metadata base.

Reply:
I don't think there is a base url field per `ogp.me` spec. I'll add the full URL with hostname

Reply:
I updated the OG image to be 1:1 as well (this is recommended).